packages/squiggle-lang/__tests__/helpers/reducerHelpers.ts

```diff
@@ -66,8 +66,12 @@ export async function expectEvalToMatch(
   expect(resultToString(await evaluateStringToResult(code))).toMatch(expected);
 }
 
-export function testEvalToBe(expr: string, answer: string) {
-  test(expr, async () => await expectEvalToBe(expr, answer));
+export function testEvalToBe(expr: string, answer: string, only = false) {
+  if (only) {
```

Comment:
I don't mind this, but another way to achieve the same goal is to pass `-t REGEX` to `pnpm test`. Also can be combined with a single file, `pnpm test __tests__/my_test.ts -- -t "my test"`.

Reply:
Good point, thanks!